### PR TITLE
fix: fix send max for eth bug

### LINKED ISSUE #239: Set default trading pairs to ETH/FOX
Set the base trading pairs to ETH/FOX

### PR DESCRIPTION
**Pull request by DaoDev44**
- Pads the estimated gas fee for max ETH trades. 
- Changes default trading to ETH/FOX

closes #239

### CODE REVIEW

**Review by elmutt — APPROVED** [review]

**Review by elmutt — APPROVED** [review]

File: packages/swapper/src/swappers/zrx/getSendMaxAmount/getSendMaxAmount.test.ts [hunk 1]
```diff
@@ -2,10 +2,15 @@ import { HDWallet } from '@shapeshiftoss/hdwallet-core'
 import { chainAdapters } from '@shapeshiftoss/types'
 import BigNumber from 'bignumber.js'
 
+import { ETH_FEE_ESTIMATE_PADDING } from '../utils/constants'
 import { setupQuote } from '../utils/test-data/setupSwapQuote'
 import { chainAdapterMockFuncs, setupZrxDeps } from '../utils/test-data/setupZrxDeps'
 import { getSendMaxAmount } from './getSendMaxAmount'
 
+const createPaddedFee = (value: string) => {
+  return new BigNumber(value).times(ETH_FEE_ESTIMATE_PADDING).toString()
```
Comment (cjthompson) [inline packages/swapper/src/swappers/zrx/getSendMaxAmount/getSendMaxAmount.test.ts:11]: guard?

File: packages/swapper/src/swappers/zrx/getSendMaxAmount/getSendMaxAmount.ts [hunk 2]
```diff
@@ -48,15 +49,19 @@ export async function getSendMaxAmount(
...
     }
   })
 
   const estimatedFee = feeEstimates[feeEstimateKey].txFee
-  const sendMaxAmount = new BigNumber(balance).minus(estimatedFee)
+
+  // (ryankk) We need to pad the fee for ETH max sends because the fee estimate is based off
+  // of a minimum quote value (quote.sellAmount) and not the users full ETH balance.
+  const paddedFee = new BigNumber(estimatedFee).times(ETH_FEE_ESTIMATE_PADDING)
+  const sendMaxAmount = new BigNumber(balance).minus(paddedFee)
```
Comment (cjthompson) [inline packages/swapper/src/swappers/zrx/getSendMaxAmount/getSendMaxAmount.ts:64]: guards?

File: packages/swapper/src/swappers/zrx/getSendMaxAmount/getSendMaxAmount.ts [hunk 3]
```diff
@@ -48,17 +49,22 @@ export async function getSendMaxAmount(
```
Comment (DaoDev44) [inline packages/swapper/src/swappers/zrx/getSendMaxAmount/getSendMaxAmount.ts:49]: add guard clause to check for quote and quote.sellAsset

File: packages/swapper/src/swappers/zrx/getSendMaxAmount/getSendMaxAmount.ts [hunk 4]
```diff
@@ -48,17 +49,22 @@ export async function getSendMaxAmount(
 
   const feeEstimates = await adapter.getFeeData({
     to: quote.depositAddress,
-    value: balance,
+    value: quote.sellAmount,
```
Comment (DaoDev44) [inline packages/swapper/src/swappers/zrx/getSendMaxAmount/getSendMaxAmount.ts:52]:
```
bnOrZero(quote.sellAmount).toString()
```

**Review by cjthompson — APPROVED** [review]

**Pull request merged**